mbbsd/vote.c

```diff
@@ -966,6 +1144,48 @@ user_vote_one(const vote_buffer_t *vbuf, const char *bname)
 		    }
 		}
 		move(b_lines - 1, 0);
+
+		/* log this vote */
```

Comment:
Please move this into a function as well. I suggest
`static int maybe_log_vote(const vote_logconf_t *logconf, const char *chosen, int nitems, const char *logfile);`